Makefile.am

```diff
@@ -47,6 +47,7 @@ linux_platform_headers = \
 	linux/LinuxProcessList.h \
 	linux/LinuxCRT.h \
 	linux/Battery.h \
+	TemperatureMeter.h \
```

Comment:
This should probably go under `myhtopheaders`

Reply:
Same as in comment below.

Reply:
Nevermind, managed to add --enable-sensors.